src/main/java/UI/ListingListPage.java

```diff
@@ -1,36 +1,131 @@
 package UI;
 import entities.Listing;
+import database.DatabaseController;
 
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
+import javax.swing.JFrame;
+import javax.swing.*;
 
-public class ListingListPage extends Page {
-    private List<Listing> listings;
+import static javax.swing.SwingUtilities.paintComponent;
+
+public class ListingListPage extends JFrame implements ActionListener {
```

Comment:
Weren't we doing ListingListPage extends JPanel?

Reply:
i don't know. this wasn't even supposed to be in this pr. we will fix later if it is wrong